quick step by step update (#2394)
saucelabs/sauce-docs: docs/mobile-apps/mdm-support.md

**Retained App Data**
All apps can be retained on the devices of your choice, and the Apple ID used for App Store login can also be retained on the device.

## Installing MDM

1. Start any Live Testing session with the private device where you want to install your MDM.
2. Sign in with your AppleID and download Intune or install your MDM using a specific link.
3. Login with your service account to your MDM/Intune portal.

> 4. After successful login, download and install the MDM profile through settings.

Reviewer: ```suggestion
4. After successful login, download and install the MDM profile through **Settings**.
```

> 5. Navigate to Settings/VPN and Device Management, click on Management Profile, and install the MDM.

Reviewer: ```suggestion
5. Navigate to **Settings > VPN and Device Management**, click on **Management Profile**, and install the MDM.
```

> 6. Once installed, click on the Home Screen.

Reviewer: ```suggestion
6. Once installed, click on the **Home Screen**.
```

7. Confirm that all the apps related to your company-managed profile have been successfully downloaded.

> ## Retrieving the BundleID for allowlisting your apps

Reviewer: ```suggestion
## Retrieving the BundleID for Allow-listing Your Apps
```

> To prevent our cleaning to remove your application after every session, this guide will help you to obtain the bundleID for these applications. 

Reviewer: ```suggestion
To prevent our cleaning from removing your application after every session, this guide will help you to obtain the bundleID for these applications. 
```

Follow these steps during live testing:

1. Open a Live Testing session with an iOS Device.
2. Open Device logs in VERBOSE mode.
3. Clean the device logs.
4. Copy-paste this string in the search bar `SBApplicationStateDisplayIDKey`.

> 5. Open the application you want to be allowlisted

Reviewer: ```suggestion
5. Open the application you want to be allow-listed.
```
